scalability_and_performance/node-observability-operator.adoc

```diff
@@ -1,13 +1,15 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="using-node-observability-operator"]
-= Requesting CRI-O and Kubelet profiling data by using the Node Observability Operator
+= Node Observability Operator
```

Comment:
Per the [OSDOCS guideline on Assembly titles](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings): 
"Try to be as descriptive as possible with the title or section headings without making them unnecessarily long. For assemblies and task modules, use a gerund form in headings"

Is it possible to add a gerund in this title since the assembly itself is a procedure type? One possibility is "Using" like another page in this section, [Using the Node Tuning Operator](https://docs.openshift.com/container-platform/4.14/scalability_and_performance/using-node-tuning-operator.html).

```suggestion
= Using the Node Observability Operator
```

Reply:
We did add gerunds for the task modules. Let me change it right away.